packages/shared/src/utils/helpers/format-response.ts

```diff
@@ -42,6 +42,40 @@ export const formatPortfolioPosition = (
     };
 };
 
+export const idv_error_statuses = Object.freeze({
+    poi_name_dob_mismatch: 'POI_NAME_DOB_MISMATCH',
+    poi_dob_mismatch: 'POI_DOB_MISMATCH',
+    poi_name_mismatch: 'POI_NAME_MISMATCH',
+    poi_expired: 'POI_EXPIRED',
+    poi_failed: 'POI_FAILED',
+});
+
+export type TIDVErrorStatus = typeof idv_error_statuses[keyof typeof idv_error_statuses];
+
+export const formatIDVError = (errors: string[], status_code: string) => {
```

Comment:
can we write a description for this function?

Reply:
alright, will leave here a comment, but anyway after BE improvements this function will be simplified or removed